cometvisu cannot be installed: The activate method has thrown an exception

Running openhabianpi, latest version 2.2.0-1.
When I try to install the cometvisu, the activation does not work, an exception is raised:

```
2018-01-24 19:16:25.638 [ERROR] [org.openhab.ui.cometvisu            ] - [org.openhab.cometvisu(272)] The activate method has thrown an exception
java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Boolean
	at org.openhab.ui.cometvisu.internal.servlet.CometVisuApp.readConfiguration(CometVisuApp.java:193) ~[?:?]
	at org.openhab.ui.cometvisu.internal.servlet.CometVisuApp.activate(CometVisuApp.java:228) ~[?:?]

```
then

```
2018-01-24 19:16:25.844 [WARN ] [org.openhab.ui.cometvisu            ] - FrameworkEvent WARNING - org.openhab.ui.cometvisu
org.osgi.framework.ServiceException: org.apache.felix.scr.impl.manager.SingleComponentManager.getService() returned a null service object
	at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse.factoryGetService(ServiceFactoryUse.java:232) ~[?:?] 
```

then again 

```
2018-01-24 19:16:25.851 [ERROR] [org.openhab.ui.cometvisu            ] - [org.openhab.cometvisu(272)] The activate method has thrown an exception
java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Boolean
	at org.openhab.ui.cometvisu.internal.servlet.CometVisuApp.readConfiguration(CometVisuApp.java:193) ~[?:?]
	at org.openhab.ui.cometvisu.internal.servlet.CometVisuApp.activate(CometVisuApp.java:228) ~[?:?] 

```

then it is uninstalled again, at least if I install it from the CLI via

`feature:install openhab-ui-cometvisu/2.2.0
`
Installing from the UI seems to create a bigger mess, at least the automatic uninstall does not work, and I had to restart openhab.
I have also deleted the existing cometvisu.cfg, but that did not help.

Any ideas what's going wrong?
Thanks
Thomas

I'm facing the same issue with the latest 2.3.0 snapshot running on a RasPi with openhabian (Linux openHABianPi 4.9.59-v7+ #1047).

When installing ui-cometvisu - 2.3.0.SNAPSHOT the follwing messages are shown at openhab.log:

`2018-01-31 08:26:35.427 [ERROR] [org.openhab.ui.cometvisu            ] - [org.openhab.cometvisu(254)] The activate method has thrown an exception
java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Boolean
        at org.openhab.ui.cometvisu.internal.servlet.CometVisuApp.readConfiguration(CometVisuApp.java:193) ~[?:?]
        at org.openhab.ui.cometvisu.internal.servlet.CometVisuApp.activate(CometVisuApp.java:228) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
........`

`2018-01-31 08:26:35.639 [WARN ] [org.openhab.ui.cometvisu            ] - FrameworkEvent WARNING - org.openhab.ui.cometvisu
org.osgi.framework.ServiceException: org.apache.felix.scr.impl.manager.SingleComponentManager.getService() returned a null service object
        at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse.factoryGetService(ServiceFactoryUse.java:232) ~[?:?]
        at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse.getService(ServiceFactoryUse.java:111) ~[?:?]
........`

`2018-01-31 08:26:35.647 [ERROR] [org.openhab.ui.cometvisu            ] - [org.openhab.cometvisu(254)] The activate method has thrown an exception
java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Boolean
        at org.openhab.ui.cometvisu.internal.servlet.CometVisuApp.readConfiguration(CometVisuApp.java:193) ~[?:?]
        at org.openhab.ui.cometvisu.internal.servlet.CometVisuApp.activate(CometVisuApp.java:228) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
........`

		
After the next refresh of the browser which is showing PaperUI the AddON-section is gone and it is showing an "ERROR: 500 - Internal Server Error".

![1](https://user-images.githubusercontent.com/18699518/35610506-ccb385f8-0661-11e8-8cb7-9513b57ec008.PNG)

![2](https://user-images.githubusercontent.com/18699518/35610516-d5041b0a-0661-11e8-888a-a38754e9ba15.PNG)

At the browsers console I can see the same error message as mentioned by ThKern.

--- a/addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/internal/backend/ConfigResource.java
+++ b/addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/internal/backend/ConfigResource.java
@@ -22,17 +22,18 @@ import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * Allows certain actions to configure the CometVisu backend trough the REST api.
+ * Allows certain actions to configure the CometVisu backend through the REST api.
  *
  * @author Tobias Bräutigam
  * @since 2.2.0
  */
-@Path(Config.COMETVISU_BACKEND_ALIAS + "/" + Config.COMETVISU_BACKEND_CONFIG_ALIAS + "/{actionName}")
+@Path(Config.COMETVISU_BACKEND_ALIAS + "/" + Config.COMETVISU_BACKEND_CONFIG_ALIAS)
 public class ConfigResource implements RESTResource {
 
     private final Logger logger = LoggerFactory.getLogger(ConfigResource.class);
 
     @GET
+    @Path("/{actionName}")
     @Produces({ MediaType.TEXT_PLAIN })
     public Response getValue(@PathParam("actionName") String actionName) {
 

--- a/addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/internal/servlet/CometVisuApp.java
+++ b/addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/internal/servlet/CometVisuApp.java
@@ -190,7 +190,7 @@ public class CometVisuApp {
                 Config.COMETVISU_WEBAPP_ALIAS = (String) properties.get(Config.COMETVISU_WEBAPP_ALIAS_PROPERTY);
             }
             if (properties.get(Config.COMETVISU_AUTODOWNLOAD_PROPERTY) != null) {
-                Boolean newValue = (Boolean) properties.get(Config.COMETVISU_AUTODOWNLOAD_PROPERTY);
+                Boolean newValue = Boolean.valueOf((String) properties.get(Config.COMETVISU_AUTODOWNLOAD_PROPERTY));
                 boolean changed = Config.COMETVISU_AUTO_DOWNLOAD != newValue;
                 Config.COMETVISU_AUTO_DOWNLOAD = newValue;
                 if (Config.COMETVISU_AUTO_DOWNLOAD && changed) {

